docs/products/opensearch/howto/os-metrics.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Aiven for OpenSearch® metrics available via Prometheus
+sidebar_label: OpenSearch metrics with Prometheus
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import HostMetrics from "@site/static/includes/host-metrics.md";
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import LimitedBadge from "@site/src/components/Badges/LimitedBadge";
+
+Monitor and optimize your Aiven for OpenSearch service with metrics available via Prometheus.
+These metrics help track cluster health, replication status, and overall performance.
+
+## Prerequisites
+
+- Enable Prometheus integration. For setup instructions, see
+  [Prometheus integration setup](/docs/platform/howto/integrations/prometheus-metrics).
+- Note the Prometheus **username** and **password** in the **Integration endpoints**
+  section of the  [Aiven console](https://console.aiven.io/).
+
+## Access Prometheus metrics
+
+<Tabs groupId="group1">
+<TabItem value="browser" label="Browser" default>
```

Comment:
Since this tab specifically shows how to access the Service URI via a browser, renaming it to "Console" might be misleading. To keep it clear and consistent with the instructions, I’ve updated the label to "View in browser."

Reply:
Ok, gotcha! Now I see that using `Browser` actually makes sense coz the steps walk you thru not only Aiven Console but also Prometheus pages so using `Console` would be misleading indeed.